python/e2e-tests/testcases/constructors.kt.py.txt

```diff
@@ -5,11 +5,11 @@ var s = "abc"
 class MultiConstructors1(a: Int) {
 
     init {
-        s = "body 11"
+        s += "|body 11"
```

Comment:
I'm surprised that `+=` operator works already :)

Reply:
Compiled like this now:

```py
def BaseClass_init__Init_(i, _this):
    global s
    BaseClass.__init__(_this)
    s = s + '|body base'
    return _this

def BaseClass_init__Create_(i):
    return BaseClass_init__Init_(i, BaseClass.__new__(BaseClass))

class BaseClass(Any):
    def equals(self, other):
        pass
    
    def hashCode(self):
        pass
    
    def toString(self):
        pass
    
    def __init__(self):
        pass
    

def ChildClass_init__Init_(a, _this):
    global s
    BaseClass_init__Init_(4, _this)
    ChildClass.__init__(_this)
    s = s + '|body child'
    return _this

def ChildClass_init__Create_(a):
    return ChildClass_init__Init_(a, ChildClass.__new__(ChildClass))

class ChildClass(BaseClass):
    def equals(self, other):
        pass
    
    def hashCode(self):
        pass
    
    def toString(self):
        pass
    
    def __init__(self):
        pass
    

def child():
    ChildClass_init__Create_(123)
    Unit_getInstance()
```